src/main/webapp/js/instructorStudentEdit.js

```diff
@@ -0,0 +1,16 @@
+$(document).ready(function(){
```

Comment:
Likewise, we should have a space after <code>function()</code> before <code>{</code> . Code should look like it's written by one person, and the indentation should at least look somewhat consistent